File: app/src/main/java/com/divercity/app/data/entity/industry/Attributes.java
package com.divercity.app.data.entity.industry;

import android.os.Parcel;
import android.os.Parcelable;

import com.google.gson.annotations.SerializedName;

public class Attributes implements Parcelable {

	@SerializedName("name")
	private String name;

	@SerializedName("description")
	private String description;

	@SerializedName("people_count")
	private int peopleCount;

	protected Attributes(Parcel in) {
		name = in.readString();
		description = in.readString();
		peopleCount = in.readInt();
	}

	@Override
	public void writeToParcel(Parcel dest, int flags) {
		dest.writeString(name);
		dest.writeString(description);
		dest.writeInt(peopleCount);
	}

	@Override
	public int describeContents() {
		return 0;
	}

	public static final Creator<Attributes> CREATOR = new Creator<Attributes>() {
		@Override
		public Attributes createFromParcel(Parcel in) {
			return new Attributes(in);
		}

		@Override
		public Attributes[] newArray(int size) {
			return new Attributes[size];
		}
	};

	public void setName(String name){
		this.name = name;
	}

	public String getName(){
		return name;
	}

	public void setDescription(String description){
		this.description = description;
	}

	public String getDescription(){
		return description;
	}

	public void setPeopleCount(int peopleCount){
		this.peopleCount = peopleCount;
	}

	public int getPeopleCount(){
		return peopleCount;
	}

	@Override
 	public String toString(){
		return 
			"Attributes{" + 
			"name = '" + name + '\'' + 
			",description = '" + description + '\'' + 
			",people_count = '" + peopleCount + '\'' + 
			"}";
		}
}

File: app/src/main/java/com/divercity/app/data/entity/location/Attributes.kt
package com.divercity.app.data.entity.location

import android.os.Parcel
import android.os.Parcelable
import com.google.gson.annotations.SerializedName

data class Attributes(

    @field:SerializedName("name")
    val name: String? = null,

    @field:SerializedName("country_name")
    val countryName: String? = null
) : Parcelable {
    constructor(parcel: Parcel) : this(
        parcel.readString(),
        parcel.readString()
    )

    override fun writeToParcel(parcel: Parcel, flags: Int) {
        parcel.writeString(name)
        parcel.writeString(countryName)
    }

    override fun describeContents(): Int {
        return 0
    }

    companion object CREATOR : Parcelable.Creator<Attributes> {
        override fun createFromParcel(parcel: Parcel): Attributes {
            return Attributes(parcel)
        }

        override fun newArray(size: Int): Array<Attributes?> {
            return arrayOfNulls(size)
        }
    }
}